MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -2578,7 +2578,7 @@ public void fixPart(IPartWork partWork, Person tech) {
         if (roll >= target.getValue()) {
             report = report + partWork.succeed();
             if (getCampaignOptions().payForRepairs()
-                    && action == " fix "
+                    && action.equals(" fix ")
```

Comment:
Wouldn't we want this as a constant?

Reply:
Yes, going for minimal changes in each area since this is an automated analysis fix-up.